_scripts/reconfigure_auth

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+## reconfigure_auth VAR_PREFIX
+## Subsume all the authentication and authorization configurations in 1 script
+
+BIN=$(dirname ${BASH_SOURCE})
+source ${BIN}/funcs.sh
+set -eo pipefail
+
+ENV_FILE=$1
+VAR_PREFIX=$2
+
+## note: should we add comments to reconfigure_htpasswd and reconfigure_oauth2 indicating they were designed to be called by this script, not independently?
```

Comment:
We can set some crazy env var in this script like __D_RY_CONFIG_ENTRY=reconfigure_auth and then if that variable is not set correctly in reconfigure_htpasswd or reconfigure_oauth2, then it aborts.

Reply:
yea I think thats the idea.

Reply:
added.